tests/cypress/integration/case_4_assign_taks_job_users.js

```diff
@@ -68,6 +68,10 @@ context('Multiple users. Assign task, job.', () => {
         it('First user login and create a task', () => {
             cy.login()
             cy.url().should('include', '/projects')
+            cy.visit('tasks')
+            if (Cypress.browser.name === 'firefox') {
```

Comment:
Can cypress define browser engine? 
This modal appears not only in Firefox actually. 
It would be better if rewrite condition like ``Cypress.browser.engine !== 'blink'``

Reply:
You can see code here: cvat-ui/src/utils/platform-checker.ts

Reply:
I'll take a look. Thank you.

Reply:
The maximum separation in the appearance of the engine only through the family
```
Cypress.browser.family !== 'chromium'
```

Reply:
Okay, let it be chromium I think.